static/app/utils/useKeyPress.tsx

```diff
@@ -1,33 +1,39 @@
-import {useEffect, useState} from 'react';
+import {RefObject, useEffect, useState} from 'react';
 
 /**
  * Hook to detect when a specific key is being pressed
  */
-function useKeyPress(targetKey: React.KeyboardEvent['key']) {
+const useKeyPress = (targetKey: string, ref?: RefObject<HTMLInputElement>) => {
```

Comment:
Should it be `RefObject<HTMLElement>`

Reply:
Yea that's fair. make this more generally usable